be/tool/beck.c

```diff
@@ -252,7 +252,7 @@ struct builder {
 	const char                *b_be_config_file; /** BE configuration */
 
 	uint64_t                   b_act;
-	uint64_t                   b_tx;
+	uint64_t                   b_data;
```

Comment:
Please add comments for these field members.

From the code, my understanding is that, this field is only used for debug log messages.